src/org.xtuml.bp.core/models/org.xtuml.bp.core/ooaofooa/External Entities/External Entities.xtuml

```diff
@@ -2333,9 +2333,13 @@ Translate:native',
         }
```

Comment:
This needs to return false at this point. If an exception is caught here then foundTree will be null and the code that follows will get an NPE.